### ISSUE TITLE
this isn't an issue just kinda like a comment

### ISSUE THREAD

**Issue by Catgenix22** [irv77/hd_fnaf#1]
we need to team up and get sister location on here if you're up for it

**Comment by irv77** [comment]
Sure. The main problem currently for me is that it doesn't detect inputs on browsers

**Comment by Catgenix22** [comment]
yes I have encountered that too, thats why i am trying to port all the files over manually

**Comment by bfdifan1** [comment]
hmm I can help testing it ig
